### DISCUSSION TITLE
Can this handle large ZIP files?

### DISCUSSION THREAD

**Discussion by Lexian-droid** [UniSharp/laravel-filemanager#1217] [Q&A]
Hello, I am working on a content management platform dedicated to application version control, and I am wondering if this package would work for my case.

I need an easier way to manage files, for each zip called a "Version" which can range from anywhere from 1mb to 30 gb, so I was wondering if it'd work for my cause, and if not, does anyone know any alternatives?

I am grateful for all help anyone can offer, thank you.

**Comment by streamtw** [comment]
Uploading files with size that big is not tested on my end.
While it may be possible to upload files as big as 30GB, it might keep users waiting too long, which results in bad user-experience.
